bad_actors/flask_api/unit_tests/flask_api_unittests.py

```diff
@@ -0,0 +1,17 @@
+import unittest, requests
+from flask_api.flask_api import add_campaign, check_status
+
+
+class TestFlaskApi(unittest.TestCase):
+
+    def add_campaign_tests(self):
```

Comment:
What you should do here, is something along the line of the following:

On setup, start the rest-api, possibly calling flask_api.py so that the rest api server will start.
Then you want to send requests to the rest api, and check you are getting the expected responses.
So for example, you can use the example requests you've created in "test_data". and send them using the python requests module.

@setup 
subprocess.start("python flask_api.py")

@test add
import requests
r = requests.post('http://localhost/api/v1/campaigns/add', data = test_data[0])
assert r.type == json
r_json = jsonfy(r.text)
assert r_json['status'] == 'success' 
assert typeof(r_json['id']) == int

@test query
r = requests.get('http://localhost/api/v1/campaigns/{0}/status'.format(saved_campaign_id),)
assert r.text in ['started working', 'waiting for machine'...] // currently this is limited

@teardown
subprocess.kill(flask_api_pid)